app/services/signature_page_service.rb

```diff
@@ -54,16 +54,19 @@ def call # rubocop:disable Metrics/AbcSize, Metrics/MethodLength
     augmented_dissertation_path
   rescue StandardError => e
     Rails.logger.error("Error generating copyright & signature pages for submission #{id}: #{e}")
-    Honeybadger.notify(e)
-    raise Error, "Failed to generate signature pages for submission #{id}: #{e.message}"
```

Comment:
This winds up notifying HB twice, which we don't need. Just re-raise the exception.